generated/nifgen/nifgen/session.py

```diff
@@ -1629,30 +1629,13 @@ def _create_waveform_f64_numpy(self, waveform_data_array):
 
         Example: :py:meth:`my_session._create_waveform_f64`
 
-        Args:
```

Comment:
this disappeared from docstring, but I don't think it should have. It's still an argument.

Reply:
This was from a metadata issue involving the `numpy` field being removed. David has another PR up to fix that now.

Reply:
Fixed!